pythonsd/tests.py

```diff
@@ -38,6 +41,43 @@ def test_coc_redirect(self):
         self.assertEqual(response.url, '/pages/code-of-conduct.html')
 
 
+class TestMeetupWidget(test.TestCase):
+    """Test the Meetup.com widget of upcoming events"""
+
+    def setUp(self):
+        self.event1 = {
+            "link": "https://example.com/event-1/",
+            "name": "Event #1",
+            "datetime": timezone.now(),
```

Comment:
Shouldn't this be `timezone.now() + datetime.timedelta(1)` so that the events are actually scheduled one day in the future? Otherwise there should be no *upcoming* events at all for either of the tests.

Reply:
Currently, these events are only used in the mock return and at no point is the datetime actually checked for anything. This is mocked so that we don't hit the meetup.com API in test.

Reply:
This ended up getting replaced anyway due to changes in order to hit the test coverage.